src/app/igreja/_components/ministries/ArticleSection.tsx

```
@@ -0,0 +1,106 @@
+import type { MinistryArticle } from "@/app/types/ministry-article";
+
+type ArticleSectionProps = {
+  articles: MinistryArticle[];
+};
+
+export default function ArticleSection({ articles }: ArticleSectionProps) {
+  return (
+    <section className="w-full py-12 md:py-16 bg-white dark:bg-gray-900">
+      <div className="max-w-7xl mx-auto px-4 md:px-8 space-y-16 md:space-y-24">
+        {articles.map((article) => {
+          const titleRef = useRef(null);
```

Biome (2.3.14)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)